src/deadline_worker_agent/sessions/session.py

```diff
@@ -1203,6 +1205,7 @@ def _handle_action_update(
             )
         )
 
+    @record_success_fail_telemetry_event(metric_name="sync_asset_outputs")  # type: ignore
```

Comment:
what mypy issue are we ignoring here?

Reply:
mypy [misc] only on OSX. It doesn't like functions with the python `**kwargs: Dict[str, Any]` type.